metaflow/cli.py

```diff
@@ -606,11 +694,124 @@ def run(obj,
+    print(f"{obj.datastore.datastore_root}\t{runtime.run_id}\tstart\t1\t_parameters\t0")
+    # runtime.execute()
+
+@parameters.add_custom_parameters
+@cli.command(help='Generate the KFP YAML to run the workflow on Kubeflow Pipelines.')
+@common_run_options
+@click.option('--output-path',
+              'pipeline_output_path',
+              default=DEFAULT_KFP_YAML_OUTPUT_PATH,
+              help="the output path (or filename) of the generated KFP pipeline yaml file")
+@click.option('--code-url',
+              'code_url',
+              default=DEFAULT_FLOW_CODE_URL,
+              help="the code URL of the flow to be executed on KFP")
+@click.option('--create-new-run',
+              show_default=True,
+              default=False,
+              help="specify whether to create a new run (Note: This option will not work as expected if your local environment is not configured to connect to the Zillow KFP cluster)"
+              )
+@click.option('--run-name',
+              'run_name',
+              default=DEFAULT_RUN_NAME,
+              help="specify whether to create a new run (Default: False) "
+                   "Note: This option will fail if your local environment is not configured to "
+                   "connect to the KFP cluster"
+              )
+@click.option('--namespace',
+              'user_namespace',
+              default=None,
+              help="Change namespace from the default (your username) to "
+                   "the specified tag. Note that this option does not alter "
+                   "tags assigned to the objects produced by this run, just "
+                   "what existing objects are visible in the client API. You "
+                   "can enable the global namespace with an empty string."
+                   "--namespace=")
+@click.pass_obj
+def run_on_kfp(obj,
+        tags=None,
+        max_workers=None,
+        max_num_splits=None,
+        max_log_size=None,
+        decospecs=None,
+        run_id_file=None,
+        user_namespace=None,
+        pipeline_output_path=DEFAULT_KFP_YAML_OUTPUT_PATH,
+        code_url=DEFAULT_FLOW_CODE_URL,
+        create_new_run=False,
+        run_name=DEFAULT_RUN_NAME,
+        **kwargs):
+    # from convert_to_kfp import create_flow_pipeline, get_ordered_steps
+    pipeline_path = create_flow_pipeline(get_ordered_steps(obj.graph), code_url, pipeline_output_path)
+    if create_new_run:
+        run_pipeline_result = kfp.Client().create_run_from_pipeline_package(pipeline_file=pipeline_output_path,
+                                                        arguments={},
+                                                        experiment_name=DEFAULT_EXPERIMENT_NAME,
+                                                        run_name=run_name)
+        print(f"\nRun created successfully!\n")
+        print(f"Run link: https://kubeflow.corp.zillow-analytics-dev.zg-int.net/pipeline/#/runs/details/{run_pipeline_result.run_id}")
```

Comment:
* let's remove references to zillow Kubeflow links.  This link might be able to be gotten from `run_pipeline_result`.

Reply:
Guessing we'd want to have a local config file and CLI args to be able to set the Kubeflow URL I'd imagine.

Reply:
Moved this out to for now. I agree that this would probably be best if configured via a local config file or provided via CLI args